app/javascript/react/components/molecules/Calendar/HeaderToggle/HeaderToggle.tsx

```diff
@@ -29,25 +37,52 @@ const HeaderToggle: React.FC<Props> = ({ titleText, componentToToggle }) => {
     };
   }, []);
 
-  const { t } = useTranslation();
+  const displayResetButton = () => {
+    if (!resetThresholds) return null;
+
+    return (
+      <S.ResetButton onClick={resetThresholds}>
+        {isMobile && (
+          <>
+            <img
+              src={returnArrow}
+              alt={t("thresholdConfigurator.altResetButton")}
+            />
+            {t("thresholdConfigurator.resetButtonMobile")}
+          </>
+        )}
+        {!isMobile && (
+          <>
+            {t("thresholdConfigurator.resetButton")}
+            <img
+              src={returnArrow}
+              alt={t("thresholdConfigurator.altResetButton")}
+            />
+          </>
+        )}
+      </S.ResetButton>
```

Comment:
I can see that this approach differs from the one that I chose. Let's talk after Majówka about which one we should continue to use because I think we should establish one rule for dealing with mobile/web.
My approach can be seen in `WeekView.style.tsx` and the use of `MobileLablel` and `DesktopLabel`

Reply:
@jankrzempek after Michaels review this code is no longer needed.

Of course, we can still talk about mobile/desktop view approach